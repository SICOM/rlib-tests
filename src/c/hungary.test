#
# This test crashes RLIB 2.0.x because it uses:
#
# rlib_set_output_parameter(r, "pdf_fontname", "Liberation Mono");
#
# RLIB 2.0.x doesn't handle fonts that are not in the 14 base fonts.
#

COMMAND=./hungary
FORMATS="pdf"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
