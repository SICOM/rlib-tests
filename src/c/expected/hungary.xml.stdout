<?xml version="1.0" encoding="ISO8859-2" ?>
<rlib>
<part>
<part_page_footer>
<output>
<line>
<data col="0" width="6" font_point="9" bold="0" italics="0" align="left" >Page: </data>
<data col="0" width="3" font_point="9" bold="0" italics="0" align="right" type="number" >  0</data>
</line>
</output>
</part_page_footer>
<part_header>
<output>
<line>
<data col="0" width="22" font_point="11" bold="0" italics="0" align="left" >ÁRVÍZTÛRÕ TÜKÖRFÚRÓGÉP</data>
</line>
<line>
<data col="0" width="22" font_point="11" bold="0" italics="0" align="left" >árvíztûrõ tükörfúrógép</data>
</line>
</output>
</part_header>
<part_page_header>
</part_page_header>
<table>
<tr>
<td width="100.000000" ><report_header>
</report_header>
<field_headers>
</field_headers>
<field_details>
</field_details>
<field_details>
</field_details>
<field_details>
</field_details>
<field_details>
</field_details>
<field_details>
</field_details>
<field_details>
</field_details>
</td>
</tr>
</table>
</part>
</rlib>
